spec/syntax.md

```diff
@@ -39,8 +39,24 @@ including message identifiers, metadata, comments, groups, etc.
 The document is part of the MessageFormat 2.0 specification,
 the successor to ICU MessageFormat, henceforth called ICU MessageFormat 1.0.
 
+### Conformance
+
+Everything in this specification is normative except for: sections marked 
+as non-normative, all authoring guidelines, diagrams, examples, and notes.
```

Comment:
Currently, `syntax.md` is structured into 5 sections:

1. Introduction
2. Overview & Examples
3. Productions
4. Tokens
5. Complete EBNF

This PR adds normative keywords to the _Overview & Examples_ section. Instead, I think only sections _3. Productions_ and _4. Tokens_ should be normative; everything else is informative.

Reply:
I think that's correct. I marked O&E as non-normative and removed keywords from it.